.github/workflows/prep_release.yml

```
@@ -0,0 +1,66 @@
+name: Prepare Viash Release
+
+on:
+  workflow_dispatch:
+    
+jobs:
+  build-and-release:
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: write
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+
+    - name: Set up JDK 17
+      uses: actions/setup-java@v4
+      with:
+        java-version: '17'
+        distribution: 'temurin'
+
+    - name: Set up sbt
+      uses: sbt/setup-sbt@v1
+
+    - name: Set up Scala
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y scala
+
+    - name: Build Viash
+      run: |
+        ./configure
+        make
+        make tools
+        bin/viash export json_schema --format=json --output=bin/schema.json
+
+    - name: Get Version
+      id: get_version
+      run: |
+        viash_version=$(bin/viash --version | cut -d ' ' -f 2)
+        echo "Detected Viash version: $viash_version"
+        echo "viash_version=$viash_version" >> "$GITHUB_OUTPUT"
+        
+        # fetch relevant changelog section
+        changelog_section=$(awk "/# Viash ${viash_version}.*/{flag=1;next}/^# /{flag=0}flag" CHANGELOG.md)
```

If you want the title to be included

```suggestion
        changelog_section=$(awk "/# Viash ${viash_version}.*/{flag=1;print;next}/^# /{flag=0}flag" CHANGELOG.md)
```